hooks/finding-post-processing/templates/NOTES.txt

```diff
@@ -0,0 +1,16 @@
+Copyright 2020 iteratec GmbH
```

Comment:
The header will also show on every installation with helm, is this desired/needed?

Reply:
I'm not sure but I think if we want to  use DCO we should include the license header (See #193 )

Reply:
Can we at least comment these out?
Should be possible here by using go templates: https://godoc.org/text/template#hdr-Actions